### ISSUE TITLE
[others] lecture assignment 11

### ISSUE THREAD

**Issue by EEE2017Assignments** [goldshakil/ElectricalProgrammingCourse2020#246]
TA,

Can I use pointer function?

For making using a pointer to have multiple return values,
I make pointer function.

double *Function(void)
{
	double* arr = (double*)malloc(sizeof(double) * 5);
	
	return arr;
}
I save x1 real value in arr[0], x1 imag vlau in arr[1], x2 real in arr[3], x2 imag in arr[4], and Determinant in arr[5].

Is this okay?

Thank you

**Comment by goldshakil** [comment]
yes ofcourse .. your function SHOULD have a pointer return value.
